components/FeatureHighlight/docs/accessibility.md

```diff
@@ -0,0 +1,24 @@
+If you would like to announce when the Feature Highlight is shown we recommend doing it inside the completion 
```

Comment:
Have we gotten approval for this use case from our internal teams? I'd like to ensure that we're happy to do this ourselves (in Google) apps if we're going to provide it as a suggestion to others.

Again, my concern is that:
1. The screen is already changing and VoiceOver does an excellent job of notifying the user.
2. The announcement is only read-out to VoiceOver users and is not provided to others, resulting in a different experience for users who use different interfaces (tapping vs. Switch Control vs. VoiceOver).

Reply:
To address your concerns:
1. VoiceOver with our current implementation doesn't notify the user that the screen has changed. It just reads the label.

2. I feel the experience should be different for VoiceOver users than for tapping or Switch Control, since tapping and switch control users can visually see that the screen changed where-as a voiceover user cannot see that the screen changed. Therefore we would need to notify them via VoiceOver that the screen changed.

Reply:
1. I double-checked `develop` and I definitely can hear the "screen changed" earcon/chirp/"boop beep" when I present the Feature Highlight in the main demo. It seems that presenting the VC is sufficient for VoiceOver to detect a screen change and communicate it to VoiceOver users.
2. I think we simply disagree about whether VoiceOver users and others should have different experiences for our components and we should seek clarification from our internal accessibility team.